assets/src/containers/CourseList.js

```diff
@@ -88,12 +87,11 @@ function CourseList (props) {
             )
             : (
               <Grid container spacing={2}>
+                <Grid item xs={12} />
                 {
                   user.relatedCourses.map((course, key) => (
                     <Grid item xs={12} sm={6} lg={4} key={key}>
-                      <Link tabIndex={-1} style={{ textDecoration: 'none' }} to={`/courses/${course.course_id}`}>
-                        <SelectCard cardData={{ title: course.course_name }} />
-                      </Link>
+                      <SelectCard cardData={{ title: course.course_name, description: course.description, path: `/courses/${course.course_id}`, enrollmentType: JSON.parse(user.enrolledCourses).filter(c => c.course_id === course.course_id)[0].enrollment_type, isAdmin: props.user.admin }} />
```

Comment:
Just a reminder that we talked about how `enrollmentType` isn't really necessary for this use of `SelectCard`, so maybe we can extract what's common to the `Course` and `CourseList` uses, or just use `Card` here with whatever linking is needed.

Reply:
[#1002 ](https://github.com/tl-its-umich-edu/my-learning-analytics/issues/1002)